docs/data-engineering/index.yml

```diff
@@ -10,8 +10,8 @@ metadata:
   ms.subservice: data-engineering 
   ms.topic: landing-page
   ms.collection: collection # Optional; Remove if no collection is used.
-  author: SnehaGunda
-  ms.author: sngun
+  author: jingzh
```

Comment:
```suggestion
  author: JeneZhang
```

Reply:
Landing and hub pages are owned by the content team. It helps our cross teams identify who owns a docset. Please revert back the author name.